doozerlib/runtime.py

```diff
@@ -281,15 +274,9 @@ def initialize(self, mode='images', clone_distgits=True,
             click.echo("Group must be specified")
             exit(1)
 
-        if self.working_dir is None:
```

Comment:
it would probably be ok, though, to fail here if no working dir had been specified?